plugin/cmd/kube-scheduler/scheduler.go

```diff
@@ -36,5 +37,8 @@ func main() {
 
 	verflag.PrintAndExitIfRequested()
 
-	app.Run(s)
+	if err := app.Run(s); err != nil {
+		glog.Fatalf("scheduler app failed to run: %v", err)
+	}
+	panic("this statement shouldn't be reached")
```

Comment:
could we change this panic to be "unreachable" to match the rest of the codebase?  It's the only outlier.